crates/common/src/opa.rs

```diff
@@ -533,6 +541,7 @@ mod tests {
     }
 
     #[tokio::test]
+    #[ignore]
```

Comment:
This is worrying. Should we add a `reason` or open a ticket or somesuch?

Reply:
This was due to rebasing onto broken .rego, now unignored

Also, in future when asserting a Result<>, just use unwrap() as it prints the error